spec/coder_test_operators.lua

```diff
@@ -24,7 +24,10 @@ local values = {
     },
     ["{integer}"] = {
         {}, {17}, {17}, {17, 18},
-    }
+    },
+    ["{x: integer, y: integer}"] = {
+        {x = 0, y = 0}, {x = 1, y = 0}, {x = 0, y = 1}, {x = 1, y = 1},
```

Comment:
I think at least one test case should appear twice in the list, like we did with arrays just above this.